templates/clusterclass/nutanix-clusterclass.yaml

```diff
@@ -0,0 +1,312 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
```

Comment:
How is this different to `templates/cluster-template-clusterclass.yaml`? The same comments apply, but I think there should only be one of these files, unless I'm missing something.

Reply:
we use kustomize to build the final templates/cluster-template-clusterclass.yaml using templates/clusterclass kustomize thru make cluster-templates command